modules/olmv1-updating-an-operator.adoc

```diff
@@ -251,26 +258,8 @@ $ oc apply -f pipelines-operator.yaml
 .Example output
 [source,text]
 ----
-clusterextension.olm.operatorframework.io/pipelines-operator configured
+clusterextension.olm.operatorframework.io/pipes configured
```

Comment:
@michaelryanpeter the name pipes is not correct with your yaml.

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/88848/files#diff-3a4664f5b5ccef6dd8762bcc6bc754af9267fa277045a6f9aeabb7807fb32087R283